wasm2c/wasm-rt-impl.c

```diff
@@ -46,23 +46,24 @@ typedef struct FuncType {
 } FuncType;
 
 #if WASM_RT_MEMCHECK_SIGNAL_HANDLER
-bool g_signal_handler_installed = false;
-char* g_alt_stack;
+static bool g_signal_handler_installed = false;
+static char* g_alt_stack;
 #else
-uint32_t wasm_rt_call_stack_depth;
-uint32_t g_saved_call_stack_depth;
+static uint32_t wasm_rt_call_stack_depth;
```

Comment:
I think this one needs to keep external linkage because it's used both in the generated output (in the function prologue/epilogue) and here in the runtime implementation. (wasm-rt.h declares it extern.)

Reply:
Hmm I guess you are correct.. i wonder how the tests passed though?

Reply:
We only use wasm_rt_call_stack_depth on platforms without the signal handler to catch OOB/exhaustion (so basically just Windows and emscripten).

Reply:
OK leaving these as public.

So we not test wasm2c under windows in CI?

Reply:
:+1: The CI does test wasm2c under Windows (and Emscripten) -- those two tests failed with the first version of the commit (https://github.com/WebAssembly/wabt/actions/runs/2660225161) but are passing now.